.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,27 @@
-### 🛠 Purpose
+### Purpose 🚀
+_clear, one sentence description of primary purpose of this PR_
 
----
+###  Notes ✏️
```

Comment:
I think you have an extra space before notes

Reply:
🦅 👁 
You're competing with @alexzherdev now for top 🕵